server/src/main/java/umm3601/user/User.java

```diff
@@ -1,7 +1,26 @@
 package umm3601.user;
 
-public class User {
+// There are two examples of suppressing CheckStyle
+// warnings in this class. If you create new classes
+// that mirror data in MongoDB and that will be managed
+// by MongoJack, then you'll probably need to suppress
+// the same warnings in your classes as well so that
+// CheckStyle doesn't shout at you and cause the build
+// to fail.
 
+// Normally you'd want all fields to be private, but
+// we need the fields in this class to be public since
+// they will be written to by Mongo via the MongoJack
+// library. We need to suppress the Visibility Modifier
+// (https://checkstyle.sourceforge.io/config_design.html#VisibilityModifier)
+// check in CheckStyle so that we don't get a failed
+// build when Gradle runs CheckStyle.
```

Comment:
We may want to change this comment, since Mongo doesn't exist yet.

```suggestion
// There are two examples of suppressing CheckStyle
// warnings in this class. If you create new classes
// that mirror data in the database and that will be managed
// by Jackson, then you'll probably need to suppress
// the same warnings in your classes as well so that
// CheckStyle doesn't shout at you and cause the build
// to fail.

// Normally you'd want all fields to be private, but
// we need the fields in this class to be public since
// they will be written to by the Jackson library. We
// need to suppress the Visibility Modifier
// (https://checkstyle.sourceforge.io/config_design.html#VisibilityModifier)
// check in CheckStyle so that we don't get a failed
// build when Gradle runs CheckStyle.
```